cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/runtime/artifact/RemoteArtifactRepositoryReader.java

```diff
@@ -39,23 +42,27 @@
 import org.apache.twill.discovery.DiscoveryServiceClient;
 import org.apache.twill.filesystem.Location;
 import org.apache.twill.filesystem.LocationFactory;
-import sun.net.www.protocol.http.HttpURLConnection;
 
+import java.io.FilterInputStream;
 import java.io.IOException;
+import java.io.InputStream;
 import java.lang.reflect.Type;
+import java.net.HttpURLConnection;
+import java.nio.charset.StandardCharsets;
 import java.util.ArrayList;
 import java.util.List;
 
-
 /**
  * Implementation for fetching artifact metadata from remote {@link ArtifactHttpHandlerInternal}
  */
 public class RemoteArtifactRepositoryReader implements ArtifactRepositoryReader {
   private static final Gson GSON = new GsonBuilder()
     .registerTypeAdapter(Schema.class, new SchemaTypeAdapter())
     .create();
-  private static final Type ARTIFACT_DETAIL_TYPE = new TypeToken<ArtifactDetail>() { }.getType();
-  private static final Type ARTIFACT_DETAIL_LIST_TYPE = new TypeToken<List<ArtifactDetail>>() { }.getType();
+  private static final Type ARTIFACT_DETAIL_TYPE = new TypeToken<ArtifactDetail>() {
+  }.getType();
```

Comment:
No need to change to newline. Just keep it as it was

Reply:
done